README.md

```diff
@@ -453,6 +453,21 @@ While most environment variables should be set in the config for a service, some
 govuk-docker run content-publisher-lite env MY_VAR=my_val bundle exec rake my_task
 ```
 
+### How to: install a new release of Ruby
+
+Many of our services use a `.ruby-version` file in conjunction with `rbenv`. When a new version of Ruby is released and we start upgrading our services, you may start seeing the following error when you run commands.
+
+```
+ruby-build: definition not found: x.y.z
+```
+
+Most of our services share a common image, which we needs rebuilding to be aware of the new Ruby version. So, to fix the error, run the following commands, where `<service>` is e.g. `collections-publisher`.
+
+```
+govuk-docker compose build --no-cache <service>-lite
+govuk-docker build
+```
+
```

Comment:
So will this process update the `.ruby-version` in a given repo?

If so is that worth stating? I always find it useful to check in on that file if I'm having issues, seeing it change might be a useful acceptance criteria for knowing this worked?

Reply:
Could you help me understand how you reached that conclusion? Perhaps it's worth pointing out the title of this section: 'install a new release of Ruby' - I don't think that implies updating anything.